### DISCUSSION TITLE
Multiple Idea requests

### DISCUSSION THREAD

**Discussion by Wxam** [polonel/trudesk#549] [Ideas]
Hey there, I have some idea's that would be great for Trudesk.

**Customers**
Have it so that we can sort customers via customer group. So that it's not just a long list of users.
If possible make it so that we don't have to fill in the e-mail adress for the user (this would be great for a General User account on the customer)

 **Tickets**
 If possible make it so that there is some sort of a timer on the Comments and Internal Notes for how long something took. this makes it easier to calculate the time after you're done with a ticket.
 
![image](https://user-images.githubusercontent.com/20145041/184620360-a4ee22e4-80b1-4692-a6d6-70e630281c2d.png)

It could be a little like this with the Time Taken on the end that you can fill in yourself.

**Comment by Wxam** [comment]
Another idea,

When changing a User account you need to set a password every single time. this is kinda frustrating when just changing some small details about the user like the group or title.

Make it so that we don't have to set a new password every single time.

**Reply by polonel** [reply]
Is this current with `1.2.5`? I just tested updating a user without changing the password and it worked correctly. 
I'm also on the `develop` branch. Can you confirm if it is working correctly for you on that branch?

**Reply by Wxam** [reply]
This is for version 1.2.5 When I try to change the user and click save account it says 
![image](https://user-images.githubusercontent.com/20145041/186110356-fcbf21cf-7d19-4e51-9f37-691d276bea55.png)

**Reply by polonel** [reply]
Try `v1.2.6`

**Reply by Wxam** [reply]
On 1.2.6 it works!
